### PR TITLE
Console log settings error.

### PR DESCRIPTION
**Pull request by abora8x8**

### CODE REVIEW

File: react/features/jaas/actionTypes.js [hunk 1]
```diff
@@ -2,3 +2,14 @@
...
+/**
+ * The type of (redux) action which signals that a specific conference was
+ * joined.
+ *
+ * {
+ *     type: CONFERENCE_JOINED,
+ *     conference: JitsiConference
+ * }
+ */
+export const CONFERENCE_JOINED = 'CONFERENCE_JOINED';
```
Comment (saghul) [inline react/features/jaas/actionTypes.js:15]: Don't re-define this, you can import it from the base/conference feature.

File: react/features/jaas/middleware.web.js [hunk 2]
```diff
@@ -19,8 +24,35 @@ MiddlewareRegistry.register(store => next => async action => {
...
+
+    case CONFERENCE_JOINED:{
+        _addErrorMsgListener(action.conference, store);
     }
     }
 
     return next(action);
 });
+
+/** .........
```
Comment (damencho) [inline react/features/jaas/middleware.web.js:38]: Can you remove these :)

File: react/features/jaas/middleware.web.js [hunk 3]
```diff
@@ -19,8 +24,35 @@ MiddlewareRegistry.register(store => next => async action => {
         if (status === STATUSES.BLOCKED) {
             store.dispatch(redirectToStaticPage('/static/planLimit.html'));
         }
+        break;
+    }
+
+    case CONFERENCE_JOINED:{
```
Comment (saghul) [inline react/features/jaas/middleware.web.js:30]: style: space after `:` also, missing `break`, which can be dangerous if we add more cases to the switch. Last, please sort the case statements alphabetically.

File: react/features/jaas/middleware.web.js [hunk 4]
```diff
@@ -19,8 +24,35 @@ MiddlewareRegistry.register(store => next => async action => {
...
+        _addErrorMsgListener(action.conference, store);
```
Comment (saghul) [inline react/features/jaas/middleware.web.js:31]: This is just a few lines, is it really worth it to extract it to a function?

**Review by saghul — COMMENTED** [review]
Left some comments, PTAL.

**Review by saghul — APPROVED** [review]

**Pull request merged**